.github/workflows/ci_cron.yml

```diff
@@ -0,0 +1,61 @@
+name: Run tests (cron)
+
+on:
+  schedule:
+    # run every Monday at 6am UTC
+    - cron: '0 6 * * 1'
```

Comment:
is this frequent enough? We used to run cron jobs daily.

Reply:
Good point. We can follow up as separate PR.